docs/customization.md

```diff
@@ -117,12 +117,22 @@ In json files, you can just override the primary keys you need. You have to over
 
 - `filter.json` to define filters to hide, their order and values (see example in https://github.com/GeotrekCE/Geotrek-rando-v3/blob/main/frontend/config/filter.json). If you want to hide some of the filter, you have to override their properties with `"display": false`.
   The `labels` filter contains an additional `withExclude` parameter. Its default value is `true`. By setting it to `true`, the user can filter the search by excluding a label (`withExclude` only works if your version of Geotrek Admin is equal to or higher than [2.77.0](https://github.com/GeotrekCE/Geotrek-admin/releases/tag/2.77.0); please set it to `false` if this is not the case)
+
 - `map.json` to define basemaps URL and attributions, center (y, x), default and max zoom level (see example in https://github.com/GeotrekCE/Geotrek-rando-v3/blob/main/frontend/customization/config/map.json).
 
-  - You can also update the map layers. There are two map layers available:
+  You can also update the map layers. Three types of map layers are available: classic, satellite and offline. Each of them is structured as follows:
+    ```ts
+      interface LayerObject {
+        url: string; // Url of the layer. It needs to be a valid tiles server url.
+        options: TileLayerOptions; // See https://leafletjs.com/reference.html#tilelayer-option
+        bounds: string; // Url of a geoJSON polygon to display this layer inside.
+      }
+    ```
+    - `mapClassicLayers`: array of `LayerObjects` for the default version.
+    - `mapSatelliteLayers`: array of `LayerObjects` for the satellite version.
+    - `mapOfflineLayer`: `LayerObject` registered for offline use. If not defined, the application uses the first layer of `mapClassicLayers` as a fallback.
```

Comment:
If not defined, it will use the one set in ``frontend/config/map.json``.
So we should rather say, If set to ``undefined`` or to ``null`` ?

Reply:
Good point! Fixed